pyproject.toml

```diff
@@ -47,6 +47,10 @@ allow_redefinition = true
 module = ["ops.*", "kubernetes.*", "flatten_json.*", "git.*"]
 ignore_missing_imports = true
 
+[[tool.mypy.overrides]]
+module = ["charms.*"]
```

Comment:
What's this for?

Reply:
For ignoring the lib's code during static analysis. I was trying to ignore it as above with an `exclude = ["lib"]` statement, but it seems the imported modules from the not excluded code are going to be analysed anyways. Let me know if that makes sense.